lib/fsevents-handler.js

```diff
@@ -89,6 +98,33 @@ function setFSEventsListener(path, realPath, listener, rawEmitter) {
   };
 }
 
+var consolidateThreshhold = 10;
+
+// Decide whether or not we should start a new higher-level
+// parent watcher
+function couldConsolidate(path) {
+  var candidates = {};
```

Comment:
Do you want the threshold variable at the top?

Also I noticed `candidates` is no longer being used.